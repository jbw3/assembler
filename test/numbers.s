# Test numbers with different bases

addi r0, 0b10101
addi r1, 0B1001
addi r2, 0o56
addi r3, 0O12
addi r4, 100
addi r5, 254
addi r6, 0123 # this is a decimal number
addi r7, 0x81
addi r8, 0XF0
