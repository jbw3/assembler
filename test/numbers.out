b015
b109
b22e
b30a
b464
b5fe
b67b
b781
b8f0
